docs/modules/user-guide/nav.adoc

```diff
@@ -1,4 +1,5 @@
 * xref:index.adoc[]
+* xref:devfile-registry.adoc[]
```

Comment:
Does its placement in the TOC look good? I thought getting started with the devfile registry would be the first step, hence, putting it right after the overview page.

Reply:
Yeah, that seems reasonable to me 👍